### DISCUSSION TITLE
"Invalid credentials" for docker-compose deployment

### DISCUSSION THREAD

**Discussion by nimishjn** [Budibase/budibase#13380] [Help]
I have specified both `BB_ADMIN_USER_EMAIL` and `BB_ADMIN_USER_PASSWORD` in my .env file, and still I am not able to login. Even when I do not specify these in `.env` file, I am not redirected to the admin page. The forgotten password is also not working.

I am locally deploying on a MacBook Pro M2.

**Comment by ddialer** [comment]
having the same issue but with deployment to EC2 instance with docker-compose

**Comment by rcarmstrong** [comment]
Had the same issue, contacted Budibase via support email, and they confirmed they could reproduce the issue so they are aware. 

To fix this I brought all containers down, unset the variables in the .env file, removed the volumes created by the docker-compose setup, and finally brought the stack back up. The key step being removing the docker volumes so there is no setup data left over from the previous try.

This fixed the issue as I was redirected to a page to create an admin user, which I could use to log in.

**Comment by ConorWebb96** [comment]
Hey all,

Theres an issue relating to this [here](https://github.com/Budibase/budibase/issues/13413). One of the engineers is currently looking at a fix for this. Any PR fixes will likely be listed against that issue. Once there is one and its released you should be able to update to the latest release and this issue will be resolved.
